Foundation/NSPathUtilities.swift

```diff
@@ -10,13 +10,22 @@
 import CoreFoundation
 
 public func NSTemporaryDirectory() -> String {
-    #if os(macOS) || os(iOS)
-    var buf = [Int8](repeating: 0, count: 100)
-    let r = confstr(_CS_DARWIN_USER_TEMP_DIR, &buf, buf.count)
-    if r != 0 && r < buf.count {
-        return String(cString: buf, encoding: .utf8)!
-    }
-    #endif
+#if os(macOS) || os(iOS)
+    var length: Int = confstr(_CS_DARWIN_USER_TEMP_DIR, nil, 0)
+    guard length > 0 else { return "/tmp" }
```

Comment:
Should this be `{ return "/tmp/" }` as it looks like trailing `/` are appended to the result.

Reply:
Yep, good catch.